js/ui.js

```diff
@@ -224,6 +224,7 @@ function replaceUI() {
 }
 
 function colorizeUser(user) {
+	if(!user)return "`C[deleted user]`";
```

Comment:
I'd like this to be consistent with the forums. They are currently
`\`C<user deleted>\``
Which is preferred?

Reply:
<user deleted> looks better to me. also on the forums so 👍

Reply:
Let's change it to `\`C<user deleted>\``

Reply:
@seanmakesgames I believe I have this change in now. Sorry didn't see this.

Reply:
no probs! Thanks for changing. :)